it/test/controllers/optOut/ConfirmOptOutControllerISpec.scala

```diff
@@ -100,12 +102,24 @@ class ConfirmOptOutControllerISpec extends ControllerISpecHelper {
 
             result should have(
               httpStatus(OK),
-              elementTextByID("heading")(optOutExpectedTitle),
-              elementTextBySelector(".govuk-summary-list__value")("2023 to 2024 tax year onwards"),
-              elementTextByID("optOut-summary")(summary),
-              elementTextByID("optOut-warning")(infoMessage),
+              elementTextByID("heading")(expectedTitle(dateService)),
+              elementTextByID("summary")(summary),
+              elementTextByID("info-message")(infoMessage),
             )
           }
+          s"throw an exception for multi-year opt out page when unsupported statuses are present" in {
+            enable(OptOutFs)
+            disable(NavBarFs)
+            stubAuthorised(mtdUserRole)
```

Comment:
Failed test here too due to the same reasons as the others. It isn't throwing an exception.